torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -556,7 +578,12 @@ def __call__(self, kernel_idx, constant_args_idx, grid, kwargs, tensors_to_clone
 
 @triton_kernel_wrapper_mutation.py_impl(DispatchKey.CompositeExplicitAutograd)
 def triton_kernel_wrapper_mutation_dense(
-    *, kernel_idx, constant_args_idx, grid, kwargs
+    *,
+    kernel_idx,
+    constant_args_idx,
+    grid,
+    tma_descriptor_metadata,
```

Comment:
Could we type hint tma_descriptor_metadata? It's difficult to tell what the format is.